src/features/views/components/ViewColumnDialog/categories.ts

```diff
@@ -0,0 +1,10 @@
+import { CHOICES } from './choices';
+
+const categories = [
+  {
+    choices: [CHOICES.FIRST_AND_LAST_NAME, CHOICES.PERSON_FIELDS],
+    key: 'basic',
```

Comment:
Maybe this key should also be an enum?

Reply:
agree! i think i thought about it and forgot